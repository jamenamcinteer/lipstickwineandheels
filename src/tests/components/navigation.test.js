import React from 'react'
import { shallow } from 'enzyme'
import Navigation from '../../components/navigation'

test('should render navigation component correctly', () => {
  const wrapper = shallow(<Navigation />)
  expect(wrapper).toMatchSnapshot()
})
