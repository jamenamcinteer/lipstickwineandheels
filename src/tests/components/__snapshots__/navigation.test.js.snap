// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navigation component correctly 1`] = `
<nav
  className="navigationContainer"
  role="navigation"
>
  <header
    className="navigationLogo"
  >
    <GatsbyLink
      to="/"
    >
      Lipstick, Wine, and Heels
    </GatsbyLink>
  </header>
  <ul
    className="navigation"
  >
    <li
      className="navigationItem"
    >
      <GatsbyLink
        to="/"
      >
        Home
      </GatsbyLink>
    </li>
    <li
      className="navigationItem"
    >
      <GatsbyLink
        to="/page/about"
      >
        About
      </GatsbyLink>
    </li>
    <li
      className="navigationItem"
    >
      <GatsbyLink
        to="/tag/"
      >
        Topics
      </GatsbyLink>
    </li>
    <li
      className="navigationItem"
    >
      <GatsbyLink
        to="/blog/"
      >
        Blog
      </GatsbyLink>
    </li>
  </ul>
</nav>
`;
